<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiscMainTab</class>
 <widget class="QWidget" name="MiscMainTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>348</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QCheckBox" name="mExcludeImportantFromExpiry">
       <property name="text">
        <string>E&amp;xclude important messages from expiry</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mLoopLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string comment="to be continued with &quot;do not loop&quot;, &quot;loop in current folder&quot;, and &quot;loop in all folders&quot;">When &amp;trying to find unread messages:</string>
       </property>
       <property name="buddy">
        <cstring>mLoopOnGotoUnread</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QWidget" name="mOnStartupOpenFolder" native="true"/>
     </item>
     <item row="1" column="1">
      <widget class="KComboBox" name="mLoopOnGotoUnread">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string comment="what's this help">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;https://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;When jumping to the next unread message, it may occur that no more unread messages are below the current message.&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Do not loop:&lt;/span&gt; The search will stop at the last message in the current folder.&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Loop in current folder:&lt;/span&gt; The search will continue at the top of the message list, but not go to another folder.&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Loop in all folders:&lt;/span&gt; The search will continue at the top of the message list. If no unread messages are found it will then continue to the next folder.&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Similarly, when searching for the previous unread message, the search will start from the bottom of the message list and continue to the previous folder depending on which option is selected.&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Loop in all marked folders: &lt;/span&gt;This is the same as &quot;Loop in all folders&quot; except that only folders are taken into account which have been marked with the folder property &quot;Act on new/unread mail in this folder&quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <item>
        <property name="text">
         <string>Do not Loop</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Loop in Current Folder</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Loop in All Folders</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Loop in All Marked Folders</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="mEnterFolderLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string comment="to be continued with &quot;jump to first new message&quot;, &quot;jump to first unread or new message&quot;, and &quot;jump to last selected message&quot;">When ente&amp;ring a folder:</string>
       </property>
       <property name="buddy">
        <cstring>mActionEnterFolder</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="KComboBox" name="mActionEnterFolder">
       <item>
        <property name="text">
         <string>Jump to First Unread Message</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jump to Last Selected Message</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jump to Newest Message</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jump to Oldest Message</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="mDelayedMarkAsRead">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Mar&amp;k selected message as read after</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="mDelayedMarkTime">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string> sec</string>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QCheckBox" name="mShowPopupAfterDnD">
       <property name="text">
        <string>&amp;Ask for action after dragging messages to another folder</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QCheckBox" name="mEmptyTrashCheck">
       <property name="text">
        <string>Empty local &amp;trash folder on program exit</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="mStartUpFolderCheck">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Open this folder on &amp;startup:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QCheckBox" name="mDeleteMessagesWithoutConfirmation">
       <property name="text">
        <string>Allow deleting messages without confirmation</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
